docs/section22/Rule22-20.md

```diff
@@ -0,0 +1,43 @@
+
+# CHW&CW - Rule 22-20  
+
+**Schema Version:** 0.0.16  
+**Mandatory Rule:** True
+**Rule ID:** 22-20  
+**Rule Description:** The tower shall be controlled to maintain a leaving water temperature, where weather permits, per Table G3.1.3.11.  
```

Comment:
Please change this to say "The tower shall be controlled to maintain a leaving water temperature, where weather permits." Compliance with the table G3.1.3.11 is verified by Rule 22-21. It's my mistake - sorry about that!